lib/prefix.c

```diff
@@ -56,6 +56,26 @@ int is_zero_mac(const struct ethaddr *mac)
 	return 1;
 }
 
+bool is_bcast_mac(struct ethaddr *mac)
+{
+	int i = 0;
+
+	for (i = 0; i < ETH_ALEN; i++) {
+		if (mac->octet[i] != 0xFF)
+			return false;
+	}
+
+	return true;
+}
+
+bool is_mcast_mac(struct ethaddr *mac)
+{
+	if ((mac->octet[0] & 0x01) == 0x01)
```

Comment:
Isn't enough to check `if (mac->octet[0] & 0x01)`? And did you think about `33-33-00-00-00-00 through 33-33-FF-FF-FF-FF ` range?

Reply:
This check accounts for that range you mentioned.